packages/ckeditor5-fullscreen/src/handlers/abstracteditor.ts

```diff
@@ -122,4 +140,70 @@ export default class AbstractEditorHandler {
 			this._container = null;
 		}
 	}
+
+	/**
+	 * Stores the current state of the annotations UIs to restore it when leaving fullscreen mode.
+	 * Code coverage is provided in the commercial package repository as integration unit tests.
```

Comment:
Could you move the info about code coverage outside the API docs, just above the Istanbul ignore?